icevision/models/interpretation.py

```diff
@@ -0,0 +1,244 @@
+__all__ = [
+    "sort_losses",
+    "get_stats",
+    "compute_weighted_sum",
+    "add_annotations",
+    "extract_losses_from_samples_dicts",
```

Comment:
What do you think about replacing `extract_losses_from_samples_dicts` by `get_samples__losses`?

Reply:
Yep, makes sense. Will do!

Reply:
Done